executor/aggfuncs/func_count.go

```diff
@@ -246,13 +246,12 @@ func (e *countOriginalWithDistinct) AppendFinalResult2Chunk(sctx sessionctx.Cont
 func (e *countOriginalWithDistinct) UpdatePartialResult(sctx sessionctx.Context, rowsInGroup []chunk.Row, pr PartialResult) (err error) {
 	p := (*partialResult4CountWithDistinct)(pr)
 
-	hasNull, isNull := false, false
 	encodedBytes := make([]byte, 0)
 	// Decimal struct is the biggest type we will use.
 	buf := make([]byte, types.MyDecimalStructSize)
 
 	for _, row := range rowsInGroup {
-		hasNull, isNull = false, false
+		var hasNull, isNull bool
```

Comment:
how about:
```suggestion
		hasNull, isNull := false, false
```

Reply:
The linter will find there is an inefficient assignment.